hack/defaults

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+TEMP_DIR=`mktemp -d`
+
+KUBEVIRT_MANIFEST_VERSION="v0.15.0"
+CDI_MANIFEST_VERSION="v1.6.0"
+
+function kubevirt_sed {
+    sed -i "s/          image: &image docker\.io\/kubevirt\/virt-operator:${KUBEVIRT_MANIFEST_VERSION}/          image: ${CONTAINER_REGISTRY}\/virt-operator:${KUBEVIRT_VERSION}/g" ${TEMP_DIR}/kubevirt-operator.yaml
```

Comment:
CONTAINER_REGISTRY variable contains `/` (a slash) which leads to error, sed can not parse such expression.

I suggest to use different character for example `|` (a pipe) instead. `sed -i "s|...|...|g`